core/common/src/MapLayerSettings.ts

```diff
@@ -280,6 +287,16 @@ export class ImageMapLayerSettings extends MapLayerSettings {
   public userName?: string;
```

Comment:
`userName`, `password`, and `accessKey` (?) probably all really belong in `unsavedQueryParams`, no?
Several APIs now take `userName`, `password`, and `queryParams` which is awkward.
Would it make sense to convert these public data members into getter/setters that read to and write from `unsavedQueryParams`?

Reply:
`userName`, `password ` are used for basic authentication, which result in a header being added to the request, not query paramters.

`accessKey` is overlapping with unsaved query parameters.  Strangely it's being overridden internally to pass the MapLayerRegistry's key inside [createImageryProvider](https://github.com/iTwin/itwinjs-core/blob/f12f5738b0739b9ccc7053d402cf5776c7ff6b08/core/frontend/src/tile/map/MapLayerFormatRegistry.ts#L166C30-L166C31).  Such a mess...